### ISSUE TITLE
0.4.0 breaks usage with `globals: true`

### ISSUE THREAD

**Issue by hco** [thomasbrodusch/vitest-fail-on-console#9]
## Bug description
I just introduced `vitest-fail-on-console` in a project, and it did not make any tests fail even though there was console output.
As I checked the repo, I noticed that `0.4.0` was just released, and verified if 0.3.1 works - and yep, with 0.3.1 the tests fail.

If you need a repo for reproduction, let me know, I can try to set one up :)

**Comment by thomasbrodusch** [comment]
Hi, @hco thanks for your interest in `vitest-fail-on-console`!

I cannot reproduce that kind of behavior, I've just made the check according to what you've just said) using `examples/`; everything seems to work with `globals: true` and no `globals` in `vitest.config.js`. 

Can you provide me a repo to have a look at it (maybe there is a mismatch in your configuration or something)

**Comment by JarvisH** [comment]
It works when importing the relevant files directly from the "src" folder (like in the examples), but when importing from node modules it does not work:

```js
import failOnConsole from 'vitest-fail-on-console'

// does nothing
failOnConsole()
```

**Comment by jamesdh** [comment]
Just ran into this as well. v0.3.1 works as expected, v0.4.1 does not detect the console warnings. 

Related to #8

**Comment by thomasbrodusch** [comment]
Hi @hco @JarvisH  and @jamesdh, thanks for your insights about this issue.
I've just published a patch `v0.4.2` available on NPM.

Have a try and tell me if everything is OK on your side!
Thanks a lot to use [vitest-fail-on-console](https://github.com/thomasbrodusch/vitest-fail-on-console)
